manifests/plugin/disk.pp

```diff
@@ -17,9 +17,9 @@
       $_manage_package = $manage_package
     } else {
       if versioncmp($collectd::collectd_version_real, '5.5') >= 0
-        and versioncmp($facts['os']['release']['major'],'8') >= 0 {
+      and versioncmp($facts['os']['release']['major'],'8') >= 0 {
```

Comment:
This also looks incorrect. I think that's a bug in the actual lint plugin on multi line if conditions. Perhaps it needs braces?

Reply:
yes it need braces. I will add them and raise an issue upstream

Reply:
not sure if this an actual bug in the Puppet DSL. the code still compiles correctly?

Reply:
AFAIK Puppet DSL doesn't care about whitespace at all.